### PR TITLE
release for September 2019

### PR DESCRIPTION
**Pull request by erich-wang**
## Checklist

- [ ] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Compute/Compute/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
```
Comment (markcowl) [inline src/Compute/Compute/ChangeLog.md]: I don't see an update in Az.Compute.psd1 to accompany this.

File: src/ApiManagement/ApiManagement/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+## Version 1.3.1
```
Comment (markcowl) [inline src/ApiManagement/ApiManagement/ChangeLog.md]: I don't see any changes in the accompanying .psd1 file for any of the modules

File: tools/RunVersionController.ps1 [hunk 3]
```diff
@@ -151,11 +151,11 @@ function Update-ChangeLog
     ($Content + $ChangeLogContent) | Set-Content -Path $ChangeLogFile.FullName -Encoding UTF8
 }
 
-if (!(Test-Path "C:/Program Files/PowerShell/Modules/PowerShellGet"))
+if (!(Test-Path "C:/Program Files/PowerShell/6/Modules/PowerShellGet"))
```
Comment (markcowl) [inline tools/RunVersionController.ps1]: I think you may need to run this from Windows PowerShell, for now, this *could* be the source of the psd1 issue.

**Pull request closed without merging**